op-node/rollup/types_test.go

```diff
@@ -233,14 +233,15 @@ func TestActivations(t *testing.T) {
 			},
 		},
 		{
-			name: "Granite",
+			name: "Holocene",
 			setUpgradeTime: func(t *uint64, c *Config) {
-				c.GraniteTime = t
+				c.HoloceneTime = t
 			},
 			checkEnabled: func(t uint64, c *Config) bool {
-				return c.IsGranite(t)
+				return c.IsHolocene(t)
 			},
 		},
+
```

Comment:
Holocene should be _added_ here, not replace the Granite case.

Reply:
oops, copy paste error (as in I obviously neglected to paste). thanks for catching. fixed.